Readme improvements (#43)
qonto/prometheus-rds-exporter: README.md

- Logs size
- RDS quota usage information

> [!TIP]

> > Prometheus RDS exporter is part of the [Database Monitoring Framework](https://github.com/qonto/database-monitoring-framework).
> > Take a look at the project to get alerts and runbooks for AWS RDS.

Reviewer: ```suggestion
> Prometheus RDS exporter is part of the [Database Monitoring Framework](https://github.com/qonto/database-monitoring-framework) which provides alerts, along with their handy runbooks for AWS RDS.
```
Nitpick

Author: I applied the changes.
